src/test/rustdoc/macros_2.rs

```diff
@@ -0,0 +1,16 @@
+
+#![feature(decl_macro)]
+
+// @has macros_2/macro.my_macro.html //pre 'pub macro my_macro() {'
+// @has - //pre '...'
+// @has - //pre '}'
+pub macro my_macro() {
+
+}
+
+// @has macros_2/macro.my_macro_2.html //pre 'pub macro my_macro_2($($tok:tt)*) {'
+// @has - //pre '...'
+// @has - //pre '}'
+pub macro my_macro_2($($tok:tt)*) {
+
+}
```

Comment:
Can you add a test with `--document-private-items` and a private macro? See https://rustc-dev-guide.rust-lang.org/tests/adding.html#header-commands-configuring-rustc.

Reply:
Will do. Thank you for your help, and patience, in reviewing these changes.

Reply:
You're very welcome! Thank you for fixing this :heart: